app/controllers/users/invitations_controller.rb

```diff
@@ -14,10 +14,15 @@ def new
     end
 
     def create
-      user_for_authorisation = User.new(invite_params)
+      user_for_authorisation
       authorize user_for_authorisation
-
-      super
+      if user_for_authorisation.deleted?
+        flash[:alert] = t('devise.invitations.user_exists', email: Settings.mail.tech_support)
+        @user = user_for_authorisation
+        render :new
```

Comment:
I think rather than rendering the `:new` template, we should redirect back to the `/new` action, to prevent re-submission?